paymentsheet/src/main/java/com/stripe/android/customersheet/data/CustomerSessionInitializationDataSource.kt

```diff
@@ -37,6 +48,10 @@ internal class CustomerSessionInitializationDataSource @Inject constructor(
                     ),
                     savedSelection = savedSelection,
                     permissions = CustomerPermissions(
+                        canRemoveLastPaymentMethod = when (configuration.allowsRemovalOfLastSavedPaymentMethod) {
```

Comment:
nit: I think you can write this as configuration.allowsRemovalOfLastSavedPaymentMethod && canRemoveLastPaymentMethodFromCustomerSession